qipr_approver/approver/forms/about_you_form.py

```diff
@@ -61,3 +61,17 @@ def __init__(self, user=User()):
                             'options': [item.name for item in Suffix.objects.all()],
                             'selected': [item.name for item in user.person.suffix.all()]}
 
+        self.last_login = {'name': 'last_login',
+                           'label': 'Last Login',
+                           'type': 'text',
+                           'value': user.person.last_login_time or ''}
```

Comment:
What do these time text inputs look like in the app? There exists a "datetime" type for inputs [here](http://www.w3schools.com/html/html_form_input_types.asp)